planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -1933,26 +1934,30 @@ void AvoidanceModule::generateExpandDrivableLanes(BehaviorModuleOutput & output)
 
 PathWithLaneId AvoidanceModule::extendBackwardLength(const PathWithLaneId & original_path) const
 {
+  const auto previous_path = helper_.getPreviousReferencePath();
+
   // special for avoidance: take behind distance upt ot shift-start-point if it exist.
```

Comment:
up to?

Reply:
Oh. typo.

Reply:
And, it's NOT special logic for avoidance  actually.

Reply:
Remove misreading comment in https://github.com/autowarefoundation/autoware.universe/pull/5046/commits/b6354f638f70104e5d05aab64a6c200aaadd3b7a.